<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>764</width>
    <height>440</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>阿里巴巴普惠体 2.0 65 Medium</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>主菜单</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLineEdit {
    border-width: 1px;
    border-style: solid;
    border-radius: 4px;
}</string>
  </property>
  <widget class="QTextBrowser" name="ShowToday">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>251</width>
     <height>341</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>阿里巴巴普惠体 2.0 65 Medium</family>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="Start_only">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>80</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>阿里巴巴普惠体 2.0 65 Medium</family>
    </font>
   </property>
   <property name="text">
    <string>开始-单发</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Start_batch">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>160</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>阿里巴巴普惠体 2.0 65 Medium</family>
    </font>
   </property>
   <property name="text">
    <string>开始-批量</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mode">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>240</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>阿里巴巴普惠体 2.0 65 Medium</family>
    </font>
   </property>
   <property name="text">
    <string> 模板 - 卡池</string>
   </property>
  </widget>
  <widget class="QPushButton" name="about">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>320</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>阿里巴巴普惠体 2.0 65 Medium</family>
    </font>
   </property>
   <property name="text">
    <string>关于</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="ShowHistory">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>90</y>
     <width>251</width>
     <height>341</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>阿里巴巴普惠体 2.0 65 Medium</family>
    </font>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>251</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>阿里巴巴普惠体 2.0 65 Medium</family>
    </font>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'阿里巴巴普惠体 2.0 65 Medium'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:36pt; font-weight:600;&quot;&gt;今日榜单&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>251</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>阿里巴巴普惠体 2.0 65 Medium</family>
    </font>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'阿里巴巴普惠体 2.0 65 Medium'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:36pt; font-weight:600;&quot;&gt;历史总榜&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
